embedded-graphics/src/mock_display.rs

```diff
@@ -238,6 +238,34 @@ where
         self.pixels[i as usize] = color;
     }
 
+    /// Get the affected area of the display.
+    pub fn affected_area(&self) -> Rectangle {
+        let (tl, br) = self
+            .pixels
+            .iter()
+            .enumerate()
```

Comment:
Would it be easier to get the point by using `self.bounding_box().points().zip(self.pixels.iter())`?